src/langgraph-platform/hybrid.mdx

```diff
@@ -34,4 +34,50 @@ For information on how to deploy a [LangGraph Server](/langgraph-platform/langgr
 
 <Tip>
   If you would like to deploy to Kubernetes, you can follow the [Hybrid deployment guide](/langgraph-platform/deploy-hybrid).
-</Tip>
\ No newline at end of file
+</Tip>
+
+## Listeners
+
+In a hybrid deployment, one or more ["listener" applications](/langgraph-platform/data-plane#”listener”-application) can be deployed depending on the organization of LangSmith workspaces and Kubernetes clusters.
+
+**Kubernetes cluster organization**
+- One or more listeners can be deployed on a Kubernetes cluster.
+- A listener can be configured to deploy to one or more Kubernetes namespaces in the cluster.
+- It's the responsibility of the owner of Kubernetes to decide the optimal organization of listeners for their use case. This requires planning ahead of time for how LangGraph Server deployments should be organized.
+
+**LangSmith workspace organization**
+- A workspace can have one or more listeners associated with it.
+- LangGraph Server deployments in a workspace can only be deployed to Kubernetes clusters where each of the listeners of the workspace is deployed to.
+
+### Use Cases
+
+The following sections describe use cases for how listeners can be configured depending on the organization of LangSmith workspaces and Kubernetes clusters. However, these not hard requirements. They are a non-exhaustive list of examples.
+
+**Each LangSmith workspace deploys to a separate Kubernetes cluster**
```

Comment:
If we think that we could be linking customers to these individual sections based on their needs, I'd consider making them H4 headers for each of linking. (I know bolded example sentences are quite long and it might not look as pretty). I could go either way here, but that possibility came to mind.

Reply:
I agree linking to them will be nice. It looks bulky as H4 the navigation, but I think utility trumps looks in this case.